css-four-column-layout/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS-four-column</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@300;500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+
+    <div class="container">
+
+      <div class="row">
+        <div class="column-full">
+          <h1>How It Works</h1>
+        </div>
+      </div>
+
+      <div class="row phone-pictures">
+        <div class="column-quarter">
+          <img class="phone" src="images/screen-1.png" alt="screen-1">
+        </div>
+
+        <div class="column-quarter">
+          <img class="phone" src="images/screen-2.png" alt="screen-2">
+        </div>
+
+        <div class="column-quarter">
+          <img class="phone" src="images/screen-3.png" alt="screen-3">
+        </div>
+
+        <div class="column-quarter">
+          <img class="phone" src="images/screen-4.png" alt="screen-4">
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="column-quarter icons">
+          <img class="switch" src="images/switch-icon.png" alt="switch">
+          <section>
+            <h3 class="app">App Switching</h3>
+            <p class="toggle">Toggles all your apps on and off.</p>
+          </section>
+        </div>
+
+        <div class="column-quarter icons">
+          <img class="filter" src="images/filter-icon.png" alt="filter">
+          <section>
+            <h3 class="trip">Trip Filtering</h3>
+            <p class="evaluates">Evaluates your trip requests.</p>
+          </section>
+        </div>
+
+        <div class="column-quarter icons">
+          <img class="slider" src="images/slider-icon.png" alt="slider">
+          <section>
+            <h3 class="service">Service Filters</h3>
+            <p class="accepts">Accepts only your ideal trips.</p>
+          </section>
+        </div>
+
+        <div class="column-quarter icons">
+          <img class="bars" src="images/bars-icon.png" alt="bars">
+          <section>
+            <h3 class="stats">Stats + Mileage</h3>
+            <p class="tracks">Tracks your trips across all apps.</p>
+          </section>
+        </div>
+      </div>
+
+
+
+
+
+    </div>
```

Comment:
I can see why you would separate the images out into their own row from a layout class perspective, but this makes it seem like the images are unrelated to the content that follows it when reading the HTML. Instead, I would make four `column-quarter` elements, each of which contains the image, icon, subheader, and descriptive text.